presto-main/src/main/java/io/prestosql/sql/planner/LocalExecutionPlanner.java

```diff
@@ -1935,14 +1956,26 @@ private PhysicalOperation createLookupJoin(JoinNode node,
             return new PhysicalOperation(operator, outputMappings.build(), context, probeSource);
         }
 
+        private Optional<DynamicFilter> createDynamicFilter(JoinNode node,
+                                                            PlanNode probeNode, List<Symbol> probeSymbols,
+                                                            PlanNode buildNode, List<Symbol> buildSymbols,
+                                                            LocalExecutionPlanContext context)
+        {
+            if (!isEnableDynamicFiltering(context.getSession())) {
+                return Optional.empty();
+            }
+            return DynamicFilter.create(node, probeNode, probeSymbols, buildNode, buildSymbols, context.getTypes());
+        }
+
         private JoinBridgeManager<PartitionedLookupSourceFactory> createLookupSourceFactory(
                 JoinNode node,
                 PlanNode buildNode,
                 List<Symbol> buildSymbols,
                 Optional<Symbol> buildHashSymbol,
                 PhysicalOperation probeSource,
                 LocalExecutionPlanContext context,
-                boolean spillEnabled)
+                boolean spillEnabled,
+                Optional<DynamicFilter> dynamicFilter)
```

Comment:
please use `io.prestosql.sql.planner.plan.JoinNode#getDynamicFilters` instead.

Reply:
If I understood correctly, it should be done at 39a2eb47da.